dev/helm/README.md

```diff
@@ -131,8 +131,6 @@ $ helm install --name my-release -f values.yaml .
 
 ## PostgresSQL
 
-By default, PostgreSQL is installed as part of the chart.
-
```

Comment:
Why is this removed?

Reply:
Merge mistake. Good catch I'll add that back.